deploy/runtools/firesim_topology_elements.py

```diff
@@ -279,7 +279,8 @@ def copy_back_job_results_from_run(self, slotno):
                     rfsname = """/home/centos/sim_slot_{}/{}""".format(simserverindex, rfsname)
 
                 run("""sudo mount {blockfile} {mntpt}""".format(blockfile=rfsname, mntpt=mountpoint))
-                run("""sudo chmod -Rf 777 {}""".format(mountpoint))
+                run("""sudo chattr -i {}/etc/sysconfig/nfs""".format(mountpoint))
+                run("""sudo chown -R centos {}""".format(mountpoint))
```

Comment:
Information about the remote user should be factored out into a variable instead of hardcoded as `centos`, although I see the assumptions about the username and home directory being endemic to this file (so the issue is broader than this PR).

Reply:
Agreed. But as you said, probably requires a larger PR. Ill go ahead and file an issue.